config/locales/en.yml

```diff
@@ -760,10 +760,16 @@ en:
     trends:
       allow: Allow
       approved: Approved
+      confirm_allow: Are you sure to allow selected tags?
```

Comment:
```suggestion
      confirm_allow: Are you sure you want to allow selected tags?
```

Reply:
Unless I'm mistaken the syntax needs to be "Are you sure you want", if its to replace the admin.reports.are_you_sure message.